src/core/styles/_constants.scss

```diff
@@ -111,7 +114,15 @@ $font-size-s: 16px;
 $font-size-m: 18px;
 $font-size-l: 20px;
 $font-size-xl: 24px;
-$font-size-xxl: 36px;
+$font-size-2xl: 28px;
```

Comment:
I suggest using multipliers for fonts that aren't very popular in the project -
`$font-size-2m: $font-size-m * 2; // 32px`

Reply:
<img width="912" height="257" alt="image" src="https://github.com/user-attachments/assets/9f9ce564-c544-420f-a0f1-c20f54ddfe66" />
In my opinion, the previous solution was clearer and more straightforward, but I’ve updated it as you asked